app/views/incomeSources/manage/ManageSelfEmployment.scala.html

```diff
@@ -0,0 +1,176 @@
+                }
+            }</dd>
+        </div>
+        <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key">@getMessage("accounting-method")</dt>
+            <dd class="govuk-summary-list__value" id="business-accounting-method">@{
+                viewModel.businessAccountingMethod match {
+                    case Some(x) => x match {
+                        case value if value == "cash" => getMessage("cash-accounting")
+                        case value if value == "accruals" => getMessage("traditional-accounting")
+                        case _ => throw new MissingFieldException("Unexpected value for businessAccountingMethod field")
+                    }
+                    case None => messages("incomeSources.generic.unknown")
+                }
+            }</dd>
+        </div>
+        @if(viewModel.itsaHasMandatedOrVoluntaryStatusCurrentYear.getOrElse(throw MissingFieldException("Missing itsaHasMandatedOrVoluntaryStatusCurrentYear field"))
+                && viewModel.latencyDetails.isDefined) {
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">@getMessage("reporting-method", viewModel.latencyDetails.get.taxYear1.toString, (viewModel.latencyDetails.get.taxYear2).toString)</dt>
+                <dd class="govuk-summary-list__value" id="reporting-method-1">@{
+                    viewModel.latencyDetails.get.latencyIndicator1 match {
+                        case method: String if method == "A" => getMessage("annually")
+                        case method: String if method == "Q" => getMessage("quarterly")
+                        case _ => "Field Missing"
+                    }
+                }</dd>
+                @if(!viewModel.taxYearOneCrystallised.get){
+                    <dd class="govuk-summary-list__actions govuk-!-text-align-right govuk-!-margin-0" id="change-box-1">
+                        <a class="govuk-link" id="change-link-1" href=@changeReportingMethodUrl(viewModel.incomeSourceId, taxYearValue(viewModel.latencyDetails.get.taxYear1), changeToValue(viewModel.latencyDetails.get.latencyIndicator1))>
+                        @getMessage("change")
+                        </a>
+                    </dd>
+                }
+            </div>
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">@getMessage("reporting-method", viewModel.latencyDetails.get.taxYear2.toString, (viewModel.latencyDetails.get.taxYear2 + 1).toString)</dt>
+                <dd class="govuk-summary-list__value" id="reporting-method-2">@{
+                    viewModel.latencyDetails.get.latencyIndicator2 match {
+                        case method: String if method == "A" => getMessage("annually")
+                        case method: String if method == "Q" => getMessage("quarterly")
+                        case _ => "Field Missing"
```

Comment:
same, as above, you need to get this value from resource / use getMessage

Reply:
going to remove line